xwiki-platform-core/xwiki-platform-eventstream/xwiki-platform-eventstream-default/src/main/java/org/xwiki/eventstream/internal/UntypedEventListener.java

```diff
@@ -70,15 +70,20 @@
      */
     public static final String EVENT_BINDING_NAME = "event";
 
+    /**
+     * The binding name of the event source when a validation expression is rendered.
+     */
+    public static final String SOURCE_BINDING_NAME = "source";
+
     @Inject
     private ObservationManager observationManager;
 
     @Inject
     private TemplateManager templateManager;
 
     @Inject
-    @Named("plain/1.0")
-    private BlockRenderer plainRenderer;
+    @Named("html/5.0")
```

Comment:
This has nothing to do with velocity. But plain renderer skip RawBlock indeed (what you end up when you use the html macro for example).